mvc/controllers/actions.rst

```diff
@@ -1,8 +1,75 @@
+
+By convention, controller classes:
+	* are located in the root-level "Controllers" folder
+	* inhert from Microsoft.AspNet.Mvc.Controller
+
+These two conventions are not required.
+
+Within the Model-View-Controller pattern, a Controller is responsible for the initial processing of the request and instantiation of the Model. Generally, business decisions should  be performed within the Model.
+
+.. note:: The Model should be a `Plain Old CLR Object (POCO)`, not a ``DbContext`` or database-related type.
+
+The controller takes the result of the model's processing (if any), returns the proper view along with the associated view data. Learn more: :doc:`/overview` and :doc:`/getting-started/first-mvc-app/start-mvc`.
+
+.. tip:: The Controller is a `UI level` abstraction. Its responsibility is to ensure incoming request data is valid and to choose which view (or result for an API) should be returned. In well-factored apps it will not directly include data access or business logic, but instead will delegate to services handling these responsibilities.
+ 
+Defining Actions
+----------------
+Any public method on a controller type is an action. Parameters on actions are bound to request data and validated using :doc:`model binding </models/model-binding>`.
+
+.. warning:: Action methods that accept parameters should verify the ``ModelState.IsValid`` property is true.
+
+Action methods should contain logic for mapping an incoming request to a business concern. Business concerns should typically be represented as services that your controller accesses through :doc:`dependency injection <dependency-injection>`. Actions then map the result of the business action to an application state.
+
+Actions can return anything, but frequently will return an instance of ``IActionResult`` (or ``Task<IActionResult>`` for async methods) that produces a response. The action method is responsible for choosing `what kind of response`; the action result `does the responding`.
+
+Controller Helper Methods
+#########################
+
+Although not required, most developers will want to have their controllers inherit from the base ``Controller`` class. Doing so provides controllers with access to many properties and helpful methods, including the following helper methods designed to assist in returning various responses:
+
+:doc:`View </views/index>`
+	Returns a view that uses a model to render HTML. Example: ``return View(customer);``
+
+HTTP Status Code
+	Return an HTTP status code. Examples: ``return Ok();`` or ``return BadRequest();``
+
+Formatted Response
+	Return ``Json`` or similar to format an object in a specific manner. Example: ``return Json(customer);``
+
+Created (201)
```

Comment:
Sorry, I meant to comment on this one earlier. Created feels a bit out of place in this list, especially given you list HTTP Status Code above. How about "Content negotiated response" instead? Examples include Ok, Created, CreatedAtAction and CreatedAtRoute. 

I would also add "Redirect" as another set of helpers. Examples include Redirect, LocalRedirect, RedirectToAction and RedirectToRoute.